bundles/http/Citrix/Employee-Survey/script.js

```diff
@@ -0,0 +1,153 @@
+const moment = library.load('moment-timezone')
+integration.define({
+    synchronizations: [
+        {
+            name: "integrationSync",
+            fullSyncFunction: syncFunction
+        }
+    ],
+    integrationParameters: [
+        { name: "surveyAdminAppId", type: "STRING", label: "Survey Admin App Id", required: true }
+    ],
+    model: {
+        tables: [
+            {
+                name: "survey_app_data",
+                columns: [
+                    { name: "response_app_id", type: "INTEGER", primaryKey: true },
+                    { name: "app_item_id", type: "INTEGER", primaryKey: true },
+                    { name: "name", type: "STRING", length: 255 },
+                    { name: "status", type: "STRING", length: 255 },
+                    { name: "notification_date", type: "DATE" },
+                    { name: "launch_date", type: "DATE" },
+                    { name: "expire_after", type: "DATE" },
+                    { name: "webform_link", type: "STRING", length: 255 },
+                    { name: "survey_description", type: "STRING", length: 2000 },
+                ]
+            }
+        ]
+    },
+    actions: [
+        {
+            name: "update_survey",
+            parameters: [
+                { name: "app_item_id", type: "INTEGER", required: true },
+                { name: "launch_date", type: "DATE", required: true },
+                { name: "expire_after", type: "DATE" },
+                { name: "status", type: "INTEGER", required: true }
+            ],
+            function: updateSurvey
+        }
+    ]
+})
+
+async function syncFunction({ client, dataStore, integrationParameters }) {
+    await getSurveyData({ client, dataStore, integrationParameters }, false);
+}
+
+async function validateResponse(response, surveyName) {
+    const body = await response.text()
+    if (response.headers.map['content-type']?.toLowerCase()?.includes("application/json")) {
+        const jsonBody = JSON.parse(body)
+        if (!response.ok) {
+            if (response.status === 403) {
+                throw new Error(`Unable to process the request, Invalid 'Response App Id' passed for '${surveyName}'`)
+            }
+            else {
+                console.log(JSON.stringify(jsonBody.error_description))
+                throw new Error(jsonBody.error_description)
+            }
+        } else {
+            return { responseBody: jsonBody }
+        }
+    } else {
+        console.log(body)
+        throw new Error('Unable to process the request')
+    }
+}
```

Comment:
just small nit :)
`else` branches are not needed because of `throw`
```suggestion
async function validateResponse(response, surveyName) {
    const body = await response.text()
    if (response.headers.map['content-type']?.toLowerCase()?.includes("application/json")) {
        const jsonBody = JSON.parse(body)
        if (!response.ok) {
            if (response.status === 403) {
                throw new Error(`Unable to process the request, Invalid 'Response App Id' passed for '${surveyName}'`)
            }
            console.log(JSON.stringify(jsonBody.error_description))
            throw new Error(jsonBody.error_description)
        } 
        return { responseBody: jsonBody }
    } else {
        console.log(body)
        throw new Error('Unable to process the request')
    }
}
```